memorystore/valkey/caching/snippets/src/main/java/samples/MemorystoreTTLItem.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+public final class MemorystoreTTLItem {
+
+    /** Replace the Memorystore instance id. */
+    private static final String INSTANCE_ID = "INSTANCE_ID";
+
+    /** Replace the Memorystore port, if not the default port. */
+    private static final int PORT = 6379;
+
+    /** Replace the id of the item to write to Memorystore. */
+    private static final String ITEM_ID = "ITEM_ID";
+
+    /** Replace the id of the item to write to Memorystore. */
+    private static final String ITEM_VALUE = "ITEM_VALUE";
+
+    /** Set the initial wait time for checking the cache. */
+    private static final int INITIAL_WAIT_TIME = 5000;
+
+    /** Set the final wait time for checking the cache. */
+    private static final int FINAL_WAIT_TIME = 6000;
+
+    private MemorystoreTTLItem() {
+        // No-op; won't be called
+    }
+
+    /**
+     * Writes to Memorystore with a Time-to-live(TTL) value.
+     *
+     * @param args command-line arguments
+     * @throws InterruptedException if the thread sleep is interrupted
+     */
+    public static void main(final String[] args) throws InterruptedException {
+
+        // Connect to the Memorystore instance
+        JedisPool pool = new JedisPool(INSTANCE_ID, PORT);
+
+        try (Jedis jedis = pool.getResource()) {
+
+            // Set a TTL of 10 seconds during entry creation
+            final int ttlSeconds = 10;
+            jedis.setex(ITEM_ID, ttlSeconds, ITEM_VALUE);
```

Comment:
again check return code for every jedis call?

Reply:
per discussion in earlier PR. setex does not return anything other than OK. so in this case we're good. let's just make sure we're checking any error codes where appropriate.